repo_policy_compliance/__init__.py

```diff
@@ -136,7 +118,7 @@ def pull_request(
 class BranchInput(BaseModel):
     """Input arguments to check jobs running on a branch.
 
-    Attrs:
+    Attributes:
         repository_name: The name of the repository to run the check on.
         branch_name: The name of the branch that the job is running on.
         commit_sha: The SHA of the commit that the job is running on.
```

Comment:
I believe these two attributes are no longer required.

Reply:
Yes, if there were to be removed though it would be a breaking change to the API contract leading to errors if they are passed so we might be stuck with them for now. We could make them optional

Reply:
They are optional now so we can remove them on the runners and them remove them from here after deploying that change

Reply:
So in terms of data validation, no error would be thrown because the validation performed by `flask_pydantic` ignores any additional fields sent by a client besides `repository_name`.

Reply:
you are right on that, I'll remove the extra fields

Reply:
Fields removed